index.html

```diff
@@ -2,18 +2,219 @@
 <hmtl>
   <head>
     <title>YouTube</title>
+    <link rel="stylesheet" href="styles/index.css" />
   </head>
   <body>
     <!-- Navbar here-->
     <header>
-      <nav class="navbar"></nav>
+      <nav class="navbar">
+        <div class="grid-item grid-item1">
+          <img class="logo" src="img/Youtubelogo.png" alt="logo" />
+        </div>
+
+        <div class="grid-item grid-item2">
+          <input type="text" name="search" placeholder="Search" />
+          <button type="button"><img id="img-btn" src="img/search.png" alt="search image" /></button>
+        </div>
+
+        <div class="grid-item grid-item3">
+          <a href="https://www.youtube.com/" class="nav_links"
+            ><img src="img/notification.png" />
+          </a>
+          <a href="#" class="nav_links"><img src="img/camera.png" /></a>
+          <a href="#" class="nav_links"><img src="img/menu.png" /></a>
+        </div>
+      </nav>
     </header>
 
     <main class="main-content">
       <!-- Main video here -->
-      <section class="video_big"></section>
+      <section class="video_big">
+        <div class="main_video">
+          <iframe
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            frameborder="0"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen
+          ></iframe>
+        </div>
+        <div class="main-text">
+          <h1>Contributing to Open Source Part I: The Easy Way</h1>
+          <div class="social-bar">
+            <div class="social-bar_text">
+              <p>59,487 views • Feb 27, 2014</p>
+            </div>
+
+            <div class="social-bar_icons">
+              <div class="icon like">
+                <img src="img/like.png" />
+                <p>440</p>
+              </div>
+
+              <div class="icon dislike">
+                <img src="img/dislike.png" />
+                <p>19</p>
```

Comment:
- [ ] This arose with this review. Try to make the details more like the originals (font-sizes, font-colors)

- [ ] Fix also the icon's alignment (share, like, dislike)

- [ ] The subscribe button needs some improvements

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/90539784-ab900180-e156-11ea-984d-30a71bed13c3.png)|![image](https://user-images.githubusercontent.com/61479648/90539950-e134ea80-e156-11ea-8f6b-3421f43f789d.png)|